### DISCUSSION TITLE
Ordered and Unordered Lists in FTD

### DISCUSSION THREAD

**Discussion by saurabh-lohiya** [fastn-stack/fastn#616] [Q&A] [answered]
What is the syntax for ordered and unordered list in FTD?

**Comment by gsalunke** [comment, marked as answer]
`ftd.text` supports markdown syntax. You can use markdown's [ordered list syntax](https://www.markdownguide.org/basic-syntax/#ordered-lists) to create ordered list:

Ordered List:

```ftd
-- ftd.text:
role: $inherited.types.copy-regular

1. Lorem ipsum dolor sit amet, consectetur
2. Lorem ipsum dolor sit amet, consectetur
3. Lorem ipsum dolor sit amet, consectetur
```
[View this online](https://fastn-stack.github.io/ftd-examples/ordered/)


Unordered List:

```ftd
-- ftd.text:
role: $inherited.types.copy-regular

- Lorem ipsum dolor sit amet, consectetur
- Lorem ipsum dolor sit amet, consectetur
- Lorem ipsum dolor sit amet, consectetur
```

[View this online](https://fastn-stack.github.io/ftd-examples/unodered/)

**Reply by saurabh-lohiya** [reply]
Thanks, this resolved my query!
